clang/include/clang/Sema/ParsedAttr.h

```diff
@@ -655,6 +655,28 @@ class ParsedAttr final
     }
   }
 
+  /// If this is an OpenCL address space attribute returns its SYCL
+  /// representation in LangAS, otherwise returns default address space.
+  LangAS asSYCLLangAS() const {
+    switch (getKind()) {
+    case ParsedAttr::AT_OpenCLConstantAddressSpace:
+      return LangAS::opencl_constant;
```

Comment:
If this is returning SYCL address spaces, should this be returning `opencl_constant`?